relion/tests/test_protocols_relion3.py

```diff
@@ -27,10 +27,10 @@
 import os
 
 import pyworkflow.tests as pwtests
+from pyworkflow.utils import copyTree, join, magentaStr
```

Comment:
join is not from pwutils, it comes from os.path. The same comment about os.path.exists applies here.